configs/rasa.json

```diff
@@ -1,32 +1,40 @@
 {
   "index_name": "rasa",
   "start_urls": [
-    "https://rasa.com/docs"
+    {
+      "url": "https://rasa.com/docs/rasa",
+      "tags": ["rasa"]
+    },
+    {
+      "url": "https://rasa.com/docs/rasa-x",
+      "tags": ["rasa-x"]
+    },
+    {
+      "url": "https://rasa.com/docs/action-server",
+      "tags": ["rasa-action-server"]
+    }
   ],
   "stop_urls": [
     "http://rasa.com/docs/.*?/\\d",
-    "http://rasa.com/docs/.*?/master",
-    ".rst.txt$",
+    "http://rasa.com/docs/.*?/next",
     "/_",
-    "/dev/",
     "[^/]$"
   ],
   "selectors": {
     "lvl0": {
-      "selector": "//div[contains(@class, 'sphinxsidebarwrapper')]//ul[contains(@class, 'current')][1]/preceding::p[contains(@class, 'caption')][1]",
-      "type": "xpath",
+      "selector": ".menu a.level--1.menu__link--active",
       "global": true,
       "default_value": "Documentation"
     },
-    "lvl1": ".body h1",
-    "lvl2": ".body h2",
-    "lvl3": ".body h3",
-    "lvl4": ".body h4",
-    "lvl5": ".body h5, .body .admonition-title",
-    "text": ".body p, .body li"
+    "lvl1": ".main h1",
+    "lvl2": ".main h2",
+    "lvl3": ".main h3",
+    "lvl4": ".main h4",
+    "lvl5": ".main h5",
+    "text": ".main p, .main li"
   },
   "selectors_exclude": [
-    ".scv-banner"
+    ".announce-bar"
```

Comment:
@lunelson would you mind double checking this config? the PR is already merged, but if there are issues I think we should address them sooner than later 🙏

Reply:
Sure, we'll have a look.

Reply:
I've made [this second PR](https://github.com/algolia/docsearch-configs/pull/2422) where I also mentioned you @Shipow :)